### PR TITLE
pc_host: added setout command line option

### PR DESCRIPTION
**Pull request by ghost**
This patch cover auto builder need of flashing and setting out a device under test for its CI flow (auto builder shall be in charge of injecting the test stuff through ssh and recollect error codes).

### CODE REVIEW

**Comment by okartau** [comment]
ok to test

File: pc_host/main.py [hunk 1]
```diff
@@ -32,13 +32,16 @@ def main():
     try:
         start_time = time.time()
         beaglebone_dut = reserve_device(args)
-        if args.emulateusb:
-            execute_usb_emulation(beaglebone_dut, args, config)
+        if args.setout:
```
Comment (skuusela) [inline pc_host/main.py:35]: Could you change it to not add new if-else structure so change it to this:
```
if args.emulateusb:
  ........
elif args.setuout:
 ........
else:
  if not args.noflash:
       ........
  if not args.notest:
```

**Review by skuusela — COMMENTED** [review]
Other than the small change this looks good to me and adds a feature that is nice to have.

**Comment by okartau** [comment]
add to whitelist

File: pc_host/main.py (hunk 1 above)
Comment (ghost) [reply pc_host/main.py:35]: Done --> https://github.com/01org/DAFT/pull/112

**Comment by jairglez** [comment]
Replaced by #112

**Pull request closed without merging**